src/components/pages/import.js

```diff
@@ -39,7 +39,8 @@ function Import({ initialState }) {
     <a
       target="_blank"
       href={getImportedArticleUrl({ hostName, allowUnsecure, slug })}
-      rel="noreferrer"
+      rel="noreferrer noreferrer"
+      noopener
```

Comment:
Something weird happened here 😬

Reply:
Done, thanks!

Reply:
Hm, I still see the same here in `src/components/pages/import-batch-view.js`

Reply:
Now it's `rel="noreferrer noreferrer"` 😂 So, it should be `rel="noreferrer noopener"`

Reply:
Fixed, sorry, got confused several times here it seems. I should pay more attention.